testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -314,6 +314,7 @@ class MiniAppDisplayActivity : BaseActivity(), PreloadMiniAppWindow.PreloadMiniA
         }
     }
 
+    //NOSONAR
```

Comment:
I'm not sure about the proper usage of this one, probably for demo app it's okay..
Do you think, it can hide the important quality flaws? 🤔

Reply:
Thanks for the catch, 

initializing bridge and its dispatchers causes to have many conditional statements that make the sonarcloud sees them as complexities. okok, I move the declaration from the function.